### PR TITLE
[Site-311] Adjust post card

### PR DESCRIPTION
**Pull request by nenispng**
**Link to issue 🔗**
https://labcodes.atlassian.net/browse/SITE-311

**What's new? ✨**
Various adjustments on the post card including:
- New fonts for title
- New separator with native font ornament font-feature
- Post cover position fixed (now properly centered)
- No more author avatar image
- Category tags are now spans instead of figcaptions
- Jello effect is more subtle now
- New hover effects 
- Color variables updated
- Post descriptions are now disabled by default

### CODE REVIEW

File: themes/medius/scss/variables.scss [hunk 1]
```diff
@@ -171,6 +166,27 @@ $white-35: rgba(255, 255, 255, .35);
...
+// Colors by category
+$business: $dark-mustard;
+$conferences: $dark-turquoise;
+$design: $dark-coral;
+$development: $dark-purple;
+
+$business-bg: $mustard;
+$conferences-bg: $turquoise;
+$design-bg: $coral;
+$development-bg: $purple;
```
Comment (fernandolins) [inline themes/medius/scss/variables.scss:187]:
```suggestion
$business: $dark-mustard;
$conferences: $dark-turquoise;
$design: $dark-coral;
$development: $dark-purple;

$business-bg: $mustard-40;
$conferences-bg: $turquoise;
$design-bg: $coral-40;
$development-bg: $purple-60;
```

You should use the right name here

File: themes/medius/scss/home.scss [hunk 2]
```diff
@@ -57,100 +58,172 @@
...
+  //   right: -$baseline;
+  //   bottom: -$baseline;
+
+  //   border-radius:  0 0 ($radius-15 - $baseline) 0;
+  //   transition: border $speed-base cubic-bezier(0.35, 1, 0.7, 1);
+  // }
+  
+  // &:focus::before {
+  //   opacity: 1;
+  // }
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:120]: Will it be useful?
otherwise, could you remove it?

File: themes/medius/scss/home.scss [hunk 3]
```diff
@@ -57,100 +58,172 @@
...
-  overflow: hidden;
+    &:hover::after {
+      @include card-hover-effect($mustard-20, $mustard-30);
+    }
 
-  @media (min-width: 768px) {
-    width: 330px;
+    // &:focus::before {
+    //   border: 2px solid $mustard-70;
+    // }
```
Comment (fernandolins) [inline themes/medius/scss/home.scss:135]: Will it be useful?
otherwise, could you remove it?

**Review by fernandolins — CHANGES_REQUESTED** [review]

File: themes/medius/scss/home.scss (hunk 2 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:120]: This code was used later on https://github.com/labcodes/blog/tree/site-292

File: themes/medius/scss/home.scss (hunk 3 above)
Comment (nenispng) [reply themes/medius/scss/home.scss:135]: This code was used later on https://github.com/labcodes/blog/tree/site-292

**Pull request merged**